packages/platform/src/lib/form-group.ts

```diff
@@ -39,6 +40,14 @@ export type FormGroupOptions = {
   disabled?: () => boolean;
 };
 
+const markFormControlAsTouched = (f: any) => {
+  if (typeof f.markAsTouched === "function") {
+    f.markAsTouched();
+  }
+  if (typeof f.markAllAsTouched === "function") {
+    f.markAllAsTouched();
+  }
```

Comment:
Small nit, I would expect `markAllAsTouched` to be called first and then `markAsTouched`.